signer/xpi/cose_test.go

```diff
@@ -507,22 +507,23 @@ func TestVerifyCOSESignaturesErrs(t *testing.T) {
 	}
 
```

Comment:
Should there be new tests around allowing expired certs?

Reply:
Yeah, I had to update the verification logic and they're buried a bit:

* the PKCS7 verification unit tests are in that PR
* `TestVerifyCOSESignaturesErrs` checks https://github.com/mozilla-services/autograph/pull/722/files#diff-c7f05b3e8291c266520101a71a41cd340754722852e691f2f05f6214be12a50cR481
  * verification at UTC now still returns an expired cert error https://github.com/mozilla-services/autograph/pull/722/files#diff-c7f05b3e8291c266520101a71a41cd340754722852e691f2f05f6214be12a50cR748
  * verification at `2019-01-01` returns a different error for invalid CN instead of an expired cert error https://github.com/mozilla-services/autograph/pull/722/files#diff-c7f05b3e8291c266520101a71a41cd340754722852e691f2f05f6214be12a50cL744-R796
* https://github.com/mozilla-services/autograph/pull/722/files#diff-59d350c2c572889a80240593671b12f226cbe506045eb575c8ddce67eb90e7bcL1170 removes a signer config with an invalid intermediate was previously expected to fail on signer init
* https://github.com/mozilla-services/autograph/pull/722/files#diff-59d350c2c572889a80240593671b12f226cbe506045eb575c8ddce67eb90e7bcR1105-R1107 adds a passing signer config testcase with the expired chain from the dev config and:
  * `TestSignFile` checks all valid configs initialize and sign a file it can verify https://github.com/mozilla-services/autograph/blob/ba655f880a8b076fbd346ef1b8215f90d8a237c7/signer/xpi/xpi_test.go#L20-L23 
* the client verifies everything it signs in the XPI integration tests https://github.com/mozilla-services/autograph/pull/722/files#diff-f71fc751da93e58e7dd840baf6efa9bd4de3c6e4126c0ce9b513776d80020130R296

Reply:
Awesome, ty!